api/envoy/api/v2/listener/listener.proto

```diff
@@ -215,3 +215,17 @@ message ListenerFilter {
     google.protobuf.Any typed_config = 3;
   }
 }
+
+message UdpListenerFilter {
+  // The name of the udp filter to instantiate. The name must match a supported
+  // filter. There are currently no built-in filters.
```

Comment:
In the final PR it would be cool to potentially `:repo:` link to the filter you write for the integration test, until if/when we have a built-in OSS one.

Reply:
Will try to have that filter started before the final PR - but in case that doesn't happen, I will leave this as it as and open an item on myself to get that done whenever it is ready.